charts/spire/charts/spire-server/templates/controller-manager-cluster-ids.yaml

```diff
@@ -24,5 +24,17 @@ spec:
   dnsNameTemplates:
     {{- toYaml . | nindent 4 }}
   {{- end }}
+  {{- with .identities.workloadSelectorTemplates }}
+  workloadTSelectoremplates:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- with .identities.ttl }}
+  ttl: {{ . | quote }}
```

Comment:
Why the extra `with` statement instead of just `ttl: {{.identities.ttl | quote }}`?

Reply:
If the ttl value is unset, it doesn't include the ttl field at all in the crd, and then spire will fall back to whatever the spire-server is set to use as a default ttl.